data/json/professions.json

```diff
@@ -1787,6 +1787,127 @@
+    "points": 4,
+    "skills": [ { "level": 3, "name": "electronics" }, { "level": 2, "name": "driving" }, { "level": 1, "name": "mechanics" } ],
+    "vehicle": "cube_van_cheap",
+    "items": {
+      "both": {
+        "items": [
+          "dress_shirt",
+          "socks",
+          "boots",
+          "pants",
+          "multitool",
+          "wristwatch",
+          "mbag",
+          "water_clean",
+          "gloves_work",
+          "hammer",
+          "hat_ball"
+        ],
+        "entries": [ { "group": "charged_cell_phone" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "id": "bus_driver",
+    "type": "profession",
+    "name": "Bus Driver",
+    "description": "You are making a living with delivering noisy brats to their schools, where you wish with every delivery, that they dont break a seat or window for you.  On your way to pick up the first row of brats, you start to notice all the chaos happening around you.  By the time you arrive at the station, all the brats look like zombies, so you took a long detour to safety, only to find yourself in a different place.  At last you have been freed of the hellish noises they release, but new ones are on the merge.",
+    "points": 4,
+    "skills": [ { "level": 1, "name": "barter" }, { "level": 3, "name": "driving" } ],
+    "vehicle": "schoolbus",
+    "items": {
+      "both": {
+        "items": [ "sneakers", "socks", "jeans", "tshirt", "wristwatch", "jacket_light", "water_clean", "hat_ball" ],
+        "entries": [ { "group": "charged_cell_phone" } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "id": "cop_car",
+    "type": "profession",
+    "name": "Police Interceptor",
+    "description": "As on-duty Officer, you received the call and got ready to respond to the robbery with your patrol car.  After arriving at the scene, it was soon you who needed rescuing, and you were lucky to escape with your life and car.  No one responds on the radio anymore either.  Who's going to respect your authority when the government this badge represents might not even exist anymore?",
+    "points": 6,
+    "skills": [ { "level": 3, "name": "gun" }, { "level": 3, "name": "pistol" }, { "level": 3, "name": "driving" } ],
+    "vehicle": "policecar",
+    "traits": [ "PROF_POLICE" ],
+    "items": {
+      "both": {
+        "items": [ "pants_army", "socks", "badge_deputy", "sheriffshirt", "police_belt", "boots", "whistle", "kevlar", "wristwatch" ],
```

Comment:
Do they have the army pants because otherwise they can't fit everything into their pockets?   I ask because police normally wear what look like slacks around here and not anything that could be remotely considered army pants.

Reply:
For some reason police wear army pants in the cataclysm, I don't know the reason but these 3 existing police profession have army pants, we could make proper police uniforms later.
https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1384
https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1441
https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1570

Reply:
> For some reason police wear army pants in the cataclysm, I don't know the reason but these 3 existing police profession have army pants, we could make proper police uniforms later.
> https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1384
> 
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1441
> 
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/f9635258544435a0c25db1320697e5924c98cb67/data/json/professions.json#L1570

I think changing police uniforms should wait for a PR similar to 49255 ...